.github/workflows/qa-deploy.yml

```diff
@@ -105,25 +105,51 @@ jobs:
 
 
   reset-db:
+    name: Reset Database
+    if: ${{ github.event.inputs.resetDb }}
```

Comment:
Does this test the existence of the `resetDb`? Or does it test the value of `resetDb`? Can we test this by running the pipeline with both options set to false?

Reply:
You were correct.  I've fixed the issue.